include/event2/http.h

```diff
@@ -598,6 +598,15 @@ EVENT2_EXPORT_SYMBOL
 void evhttp_send_reply_chunk_with_cb(struct evhttp_request *req, struct evbuffer *databuf,
     void (*cb)(struct evhttp_connection *, void *), void *arg);
 
+/**
+   Complete a chunked reply with trailers, freeing the request as appropriate.
+
+   @param req a request object
+   @param trailers the evkeyvalq object in which to find the trailers
+*/
+void
+evhttp_send_reply_end_trailers(struct evhttp_request *req, struct evkeyvalq *trailers);
```

Comment:
We definitely want to have a unit test for this

Reply:
I checked, looks like we're not even using this function at all so I don't even have an example for its usage.